source/frontend/src/models/generate/acquisition/GenerateCompensationPayee.ts

```diff
@@ -0,0 +1,52 @@
+import { Api_CompensationPayee } from 'models/api/CompensationPayee';
+import { formatMoney } from 'utils';
+import { formatNames } from 'utils/personUtils';
+
+export class Api_GenerateCompensationPayee {
+  name: string;
+  pre_tax_amount: string;
+  tax_amount: string;
+  total_amount: string;
+  gst_number: string;
+
+  constructor(payee: Api_CompensationPayee | null) {
+    if (payee?.acquisitionOwner) {
+      this.name = formatNames([
+        payee.acquisitionOwner.givenName,
+        payee.acquisitionOwner.lastNameAndCorpName,
+      ]);
+    } else if (payee?.interestHolder) {
+      if (payee.interestHolder.person) {
+        this.name = formatNames([
+          payee.interestHolder.person.firstName,
+          payee.interestHolder.person.surname,
+        ]);
```

Comment:
I noticed that in GenerateInterestHolder we use the middle-name (if present) but here we do not. Just want to confirm that is okay as per business requirements.

Also the other "persons" in this method (owner rep, owner solicitor, etc) don't have their middle-names included in the formatName() call as well.